.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy main branch
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "cds-snc-docs/**"
+
+jobs:
+  deploy:
+    name: deploy
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    
+    - name: Build
+      run: |
+       cd cds-snc-docs
+       npm ci
+       npm run build
+       mkdir ../server/build
+       cp -r ./build/* ../server/build
+    
+    - name: Serverless deploy
+      run: |
+        cd server
+        npm ci
+        node ./node_modules/serverless/bin/serverless.js config credentials --provider aws --key ${{ secrets.DEV_AWS_ACCESS_KEY_ID }} --secret ${{ secrets.DEV_AWS_SECRET_ACCESS_KEY }} --profile papaya --overwrite
+        node ./node_modules/serverless/bin/serverless.js deploy --aws-s3-accelerate --stage dev --verbose
+      env:
+         AWS_ACCESS_KEY_ID: ${{ secrets.DEV_AWS_ACCESS_KEY_ID }}
```

Comment:
You should be able to delete this and the `AWS_SECRET_ACCESS_KEY` below as well since we reference them directly from the `${{ secrets }}` object.